atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -87,6 +87,23 @@
 
     private final Random random = new Random();
 
+    @VisibleForTesting
+    public CassandraService(
```

Comment:
I can't make the other constructor call this one because it uses `this::getSnitch` in the construction of the location supplier.

Reply:
This isn't used internally (I checked internal code search), so I'd suggest breaking the API, this is kind of weird.